### ISSUE TITLE
Disable CardView who are not infront

### ISSUE THREAD

**Issue by aftabsikander** [Arjun-sna/android-swipecards-view#3]
Nice Library @Arjun-sna while going through your library I found an issue regarding cardview clickable states. Currently Card which are behind the stack are clickable, Is it possible to disable those Card which are not on Top.

**Comment by VaibhavLakhera** [comment]
@aftabsikander 

I was able to resolve it using the following way:

```
mSwipeCardView.setOnItemClickListener(new SwipeCardView.OnItemClickListener()
{
            @Override
            public void onItemClicked(int itemPosition, Object dataObject)
            {
                if (itemPosition == mSwipeCardView.getTop())
                {
                    // Perform some action here
                }
            }
});
```

**Comment by Arjun-sna** [comment]
I cannot reproduce the issue you have mention above. Only the card which is on top of stack is clickable and throws event on click. If you can reproduce it consistently please provide some more details for debugging.

**Comment by aftabsikander** [comment]
@Arjun-sna Let me rephrase it for you, I am saying cardview having custom layout which have buttons at the bottom of the view. Then if you slighty move the top cardview to any direction in resulting showing previous cardview buttons then, You can easily click them.

**For example**
If you modify your sample project layout file [item.xml](https://github.com/Arjun-sna/android-swipecards-view/blob/master/app/src/main/res/layout/item.xml) having buttons at the bottom of the view. This will easily reproduce this issue.

Only the Top view Cardview should be clickable i.e(Buttons or any other widget should be clickable) and disable all views which are behind the stack.  For now what I did to solve this issue by putting a transparent overlay view which sits in every cardview custom layout. And I modified the `SetupTopView` Method to hide this overlay for Top Card Item

**Comment by aftabsikander** [comment]
@Arjun-sna Any update on this, I have solved this issue by created a transparent view which overlaps cardview item layout, And TopView setup method is make is disappear for the current visible top card.
